.eslintrc.cjs

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  root: true,
+  env: {
+    es2022: true,
```

Comment:
node?

Reply:
Fixed in 02034a8353e1ccf5f7ccd162c8dbf1dceeafba93